cartridges/app_storefront_custom2/cartridge/models/account.js

```diff
@@ -0,0 +1,11 @@
+var base = module.superModule;
+
+function account(currentCustomer, addressModel, orderModel) {
+    base.call(this,currentCustomer,addressModel,orderModel);
```

Comment:
Please watch out for the white spaces and indentations. By convention, there should be white space after comma, equal sign, etc.

Reply:
Fixed